.goreleaser.yaml

```diff
@@ -0,0 +1,107 @@
+project_name: dnsbench
+
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+- main: ./cmd/dnsbench
+  binary: dnsbench
+  
+  env:
+  - CGO_ENABLED=0
+
+  flags:
+   - "-trimpath"
+   - "-mod=readonly"
+
+  ldflags:
+   - "-s -w"
+   - "-X github.com/askmediagroup/dnsbench/pkg/cmd.dnsbenchVersion={{ .Version }}"
+   - "-X github.com/askmediagroup/dnsbench/pkg/cmd.gitCommit={{ .Commit }}"
+   - "-X github.com/askmediagroup/dnsbench/pkg/cmd.buildDate={{ .Date }}"
+
+  goos:
+  - darwin
+  - linux
+  - windows
+
+  goarch:
+   - amd64
+   - arm64
+
+  ignore:
+    - goos: windows
+      goarch: arm64
+
+dockers:
+  - use: buildx
+    goarch: amd64
+    build_flag_templates:
+      - "--pull"
+      - "--platform=linux/amd64"
+      - "--label=org.opencontainers.image.title={{ .ProjectName }}"
+      - "--label=org.opencontainers.image.vendor=AskMediaGroup"
+      - "--label=org.opencontainers.image.description=dnsbench {{ .Version }}"
+      - "--label=org.opencontainers.image.url=https://github.com/askmediagroup/dnsbench"
+      - "--label=org.opencontainers.image.documentation=https://github.com/askmediagroup/dnsbench/blob/main/README.md"
+      - "--label=org.opencontainers.image.source=https://github.com/askmediagroup/dnsbench"
+      - "--label=org.opencontainers.image.licenses=Apache-2.0"
+      - "--label=org.opencontainers.image.version={{ .Version }}"
+      - "--label=org.opencontainers.image.revision={{ .FullCommit }}"
+      - "--label=org.opencontainers.image.created={{ time \"2006-01-02T15:04:05Z07:00\" }}"
```

Comment:
@richardmarshall What's up with that hardcoded timestamp string?

Reply:
It's a template, the goreleaser date function accepts a timestamp as an argument and parses it to figure out how you want the actual timestamp to be formatted.